frontend/src/metabase/hooks/use-safe-async-function.ts

```diff
@@ -1,12 +1,12 @@
 import { useCallback } from "react";
-import { useIsMounted } from "./use-is-mounted";
+import { useMountedState } from "react-use";
 
 type AsyncFn = (...args: any[]) => Promise<any>;
 
 // wraps the given async function in a promise that does not resolve
 // after the component has unmounted
 export function useSafeAsyncFunction(fn: AsyncFn, deps?: any[]): AsyncFn {
```

Comment:
Could you also mark this is deprecated? Really `useAsync[Fn]` should be used instead of this